packages/shared/src/hooks/useViewSize.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import { useMedia } from './useMedia';
+import { laptop, laptopL, tablet } from '../styles/media';
+
+type ViewSize = {
+  isMobile: boolean;
+  isTablet: boolean;
+  isLaptop: boolean;
+  isLaptopL: boolean;
+};
+const useViewSize = (): ViewSize => {
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
+  const isTablet = useMedia([tablet.replace('@media ', '')], [true], false);
+  const isLaptop = useMedia([laptop.replace('@media ', '')], [true], false);
+  const isLaptopL = useMedia([laptopL.replace('@media ', '')], [true], false);
```

Comment:
What about laptopXL, desktop and desktopL?

Reply:
Didn't really see usage of those for this hook. (they mostly CSS only)
As this hook can become quite heavy on window events wanted to keep it to the bare minimum.